scripts/require-sync.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Look in the library and find every `template.yaml` file
+TEMPLATE_PATHS=$(find . -name "*template.yaml")
+
+FAILURE=false
+for path in $TEMPLATE_PATHS; do
+
+  # Check for presence of 'data.inventory'
+  if ! grep -q 'data.inventory' "$path"; then
```

Comment:
This is not a 100% reliable way of detecting requiring inventory (e.g. `input.parameters.data.inventory` would trigger this as well).

More reliable would be to attempt to compile the template with `inventory` disabled in the constraint framework, but TBH I'm not sure this is a blocker.

It might be good to give `gator test` the ability to turn of inventory (@julianKatz 's old PR), as a way of easily and reliably detecting something like this via bash.

Reply:
Thanks @maxsmythe, yes it would be great if `gator test` had that capability.  In the meantime I updated the grep to `' data.inventory.'` (space at the start) which will at least avoid the possibility of a false match on `input.parameters.data.inventory`.